ext/fast_osc/fast_osc_wrapper.c

```diff
@@ -33,6 +36,8 @@ void Init_fast_osc() {
   rb_define_singleton_method(FastOsc, "decode_single_message", method_fast_osc_decode_single_message, 1);
   rb_define_singleton_method(FastOsc, "encode_single_message", method_fast_osc_encode_single_message, -1);
   rb_define_singleton_method(FastOsc, "encode_single_bundle", method_fast_osc_encode_single_bundle, -1);
+  rb_define_singleton_method(FastOsc, "is_bundle", method_fast_osc_is_bundle, 1);
```

Comment:
I think Ruby convention would prefer `is_bundle?` as the method name

Reply:
Fixed in the latest one

Reply:
@xavriley/@llloret - at least according to Rubocop style rules, the preferred would just be `bundle?`